parlai/core/dict.py

```diff
@@ -708,9 +741,13 @@ def vec2txt(self, vector, delimiter=' '):
             text = self.bpe.decode(tokens, vector, delimiter)
         elif self.tokenizer == 'bytelevelbpe':
             # We add special tokens in the beginning of ParlAI dict but in the
-            # end of Hugging Face dict,there is an offset of 4 between them.
+            # end of Hugging Face dict, there is an offset of #(extra tokens) between them.
+            extra_tokens = 4  # length of special tokens
```

Comment:
assuming this is just for clarity?

Reply:
yeah i am just making it like clear why it's 4, otherwise it seemed random to me